packages/config/src/configs/kinitsugi.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  BalanceBuilder,
+  ExtrinsicBuilder,
+  FeeBuilder,
+} from '@moonbeam-network/xcm-builder';
+import { kbtc, kint } from '../assets';
+import { kinitsugi, moonriver } from '../chains';
+import { AssetConfig } from '../types/AssetConfig';
+import { ChainConfig } from '../types/ChainConfig';
+
+export const kinitsugiConfig = new ChainConfig({
```

Comment:
typo in the name of the const and in also in the name of the file, in general replace all the `kinitsugi` with `kintsugi` in the whole project

Reply:
this is fixed in my next PR